### DISCUSSION TITLE
Can I change only certain element value of EDI?

### DISCUSSION THREAD

**Discussion by TaeWoongKim2** [xlate/staedi#307] [Q&A]
EDI is trying to encrypt certain privacy items.
In the original EDI, the item value of a specific segment needs to be encrypted and stored again, so can we implement overwriting with the reader and writer provided by this open source?

I tried the reader, and the delimiters were auto skipped...!!
How can i do this!! lol

**Comment by MikeEdgar** [comment]
Hi @TaeWoongKim2 - I understand you are dealing with sensitive data, but if there is a way you can provide an example based on mock data, that would help me understand your issue. Are the individual elements encrypted, segments, or the transaction as a whole?
